e2e/landing.spec.ts

```diff
@@ -37,136 +37,236 @@ const superBlocks = [
   intro[SuperBlocks.PythonForEverybody].title
 ];
 
-let page: Page;
+async function addDefaultCookies(context: BrowserContext, variation: string) {
+  await context.addCookies([
+    {
+      name: 'gbuuid',
+      value: variation,
+      domain: 'localhost',
+      path: '/',
+      expires: Math.floor(Date.now() / 1000) + 400 * 24 * 60 * 60 // 400 days from now
+    }
+  ]);
+}
 
-test.beforeAll(async ({ browser }) => {
-  page = await browser.newPage();
+async function goToLandingPage(page: Page) {
   await page.goto('/');
-});
+}
+
+test.describe('Landing Page - Variation B', () => {
+  test.beforeEach(async ({ context, page }) => {
+    await addDefaultCookies(context, 'B');
+    await goToLandingPage(page);
+  });
 
-test('The component Landing-top renders correctly', async () => {
-  const landingHeading1 = page.getByTestId('landing-big-heading-1');
-  await expect(landingHeading1).toHaveText(
-    translations.landing['big-heading-1']
-  );
+  test('The component Landing-top renders correctly', async ({ page }) => {
```

Comment:
It's fine, I don't think it's worth changing, but testing the page layout via playwright feels a little excessive. The way I think about it is this: we're primarily interested in knowing "is the appropriate landing page showing, given the growthbook setup + cookies?".

It's enough to know _that_ the page changes. e.g. check that `big-heading-1-b` appears when we're expecting variation B, but not when we're expecting A.

As I said, it doesn't seem worth changing, but I would recommend that in future.